src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -108,40 +111,47 @@ ov_model_reshape_input_by_name(const ov_model_t* model,
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param partialShape A PartialShape.
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_model_reshape_one_input(const ov_model_t* model, const ov_partial_shape_t* partial_shape);
+ov_model_reshape_single_input(const ov_model_t* model, const ov_partial_shape_t* partial_shape);
 
 /**
  * @brief Do reshape in model with a list of <port id, partial shape>.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param ports The port list.
  * @param partialShape A PartialShape list.
- * @param cnt The item count in the list.
+ * @param size The item count in the list.
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_model_reshape_by_ports(const ov_model_t* model, size_t* ports, const ov_partial_shape_t** partial_shape, size_t cnt);
+ov_model_reshape_by_ports(const ov_model_t* model,
+                          size_t* ports,
+                          const ov_partial_shape_t** partial_shape,
+                          size_t size);
 
 /**
  * @brief Do reshape in model with a list of <ov_output_node_t, partial shape>.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param output_nodes The ov_output_node_t list.
  * @param partialShape A PartialShape list.
- * @param cnt The item count in the list.
+ * @param size The item count in the list.
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
 ov_model_reshape_by_nodes(const ov_model_t* model,
-                          const ov_output_node_t* output_nodes[],
-                          const ov_partial_shape_t* partial_shapes[],
-                          size_t cnt);
+                          const ov_output_node_t** output_nodes,
+                          const ov_partial_shape_t** partial_shapes,
+                          size_t size);
 
 /**
  * @brief Gets the friendly name for a model.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param friendly_name the model's friendly name.
+* @return Status code of the operation: OK(0) for success.
```

Comment:
Incorrect indent. I thought that clang-format should fix it...

Reply:
Sure, It seems I have forgot checking format.

Reply:
done